INSTRUCTION.md

```diff
@@ -0,0 +1,24 @@
+# How to validate the changes
+
+#Use this command to run Bootstrap Script
+
+```
+./bootstrap.sh
+```
+
+Use this commands to verify Deployment
+
+```
+kubectl get all -n todoapp
+kubectl get all -n mysql
+```
+
+Check Ingress resource:
+
+```
+kubectl get ingress -n todoapp
+```
+
+### How to validate the changes:
+1. Access the application via http://localhost.
```

Comment:
Ensure that the application is correctly configured to be accessed via `http://localhost`, and that any necessary port forwarding or service exposure is set up.